src/cgroup.c

```diff
@@ -279,19 +279,29 @@ gboolean check_cgroup2_oom()
 	ssize_t read;
 	while ((read = getline(&line, &len, fp)) != -1) {
 		long int counter;
-
-		if (read < 6 || memcmp(line, "oom ", 4))
+		int memcmp_oom, memcmp_oom_kill, len;
```

Comment:
`memcmp_oom` and `memcmp_oom_kill` are never used, are they left over?

Reply:
Oh yeah, removed them as well as `len`, because we can re-use it from the lines above.